astro/src/content/blog/vatix-fusionauth.mdx

```diff
@@ -0,0 +1,52 @@
+---
+publish_date: 2024-10-31 
+title: Vatix Chooses FusionAuth to Streamline Authentication and Save Developer Resources
+description: What happens when your homegrown auth system is no longer sustainable?
+authors: Candace Werry
+image: /img/blogs/vatix/vatix-fusionauth.png
+tags: mfa, sso, enterprise
+categories: Community
+excerpt_separator: "{/* more */}"
+---
+
+Vatix needed a scalable, automated solution that could integrate with their enterprise customers' identity providers
+{/* more */}
+## THE CHALLENGE
+
+[Vatix](https://vatix.com/) specializes in safety and compliance software, providing customizable incident reporting solutions for enterprise organizations. As the company grew, it found itself at a crossroads with its authentication system. The needs of their clients, especially enterprise customers, expanded to require advanced authentication features.
+
+Vatix was using a homegrown authentication system that started showing limitations. It worked for a time but couldn't handle the complexity that came with a scaling business. The system required manual intervention each time Vatix onboarded a new client, creating bottlenecks that slowed down operations and made it difficult to meet customer needs.
+
+Adrian Abad, the product manager at Vatix, explained:
+
+>"Managing different authentication needs for each client had become a challenge. We needed an authentication solution that could integrate with enterprise identity providers, handle various environments, and meet diverse compliance standards."
```

Comment:
This totally depends on how you want, but you could have the quotes be normal text (no blockquote) and then have pull quotes that are not necessarily quotes.

Stuff like 

> Setting up authentication for each new client consumed many developer hours and relied heavily on one developer's availability. This manual process increased the risk of errors and potential security vulnerabilities.

is not a quote but is pretty sexy.

Same with

> Automating customer deployments has allowed Vatix to focus on what truly matters: building and improving their core product. Instead of spending valuable time managing multiple authentication systems, they can now dedicate more energy to innovation and meeting the unique needs of their customers.

Your call, that is more of a style thing.